deploy/awstools/awstools.py

```diff
@@ -39,7 +39,15 @@
 # https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;search=FPGA%20Developer;sort=name
 # And whenever this changes, you also need to update deploy/tests/test_amis.json
 # by running scripts/update_test_amis.py
-f1_ami_name = "FPGA Developer AMI - 1.12.1-40257ab5-6688-4c95-97d1-e251a40fd1fc"
+def get_f1_ami_name() -> str:
+    cuser = local("whoami", capture=True)
```

Comment:
Is there an advantage to calling out to a shell for `whoami` potentially many times vs using `env['USER']`?

Also, are you expecting that the AMI being used by the manager and runhosts needs to match?  I know this is silent support for AL2 but you might want to put a comment in this method documenting the assumption for future devs if you aren't documenting it in the actual docs.

Reply:
No advantage. I can change it to looking at `env['USER']`.

Right now that is the current assumption (I'm not sure anyone else has done otherwise). I'll add a comment.